UI/src/application.js

```diff
@@ -137,14 +140,35 @@
     new SpeedLimitWorkList().initialize();
     new InaccurateWorkList().initialize();
     new UserNotificationPopup(models.userNotificationCollection).initialize();
-
     new MunicipalitySituationPopup(models.municipalitySituationCollection).initialize();
 
     backend.getStartupParametersWithCallback(function(startupParameters) {
       backend.getAssetPropertyNamesWithCallback(function(assetPropertyNames) {
+        assetTypeId = startupParameters.assetType;
+        defaultLocation = [startupParameters.lon, startupParameters.lat, startupParameters.zoom];
+
+        assetTypeLayerName = getSelectedAssetByTypeId(pointAssets, assetTypeId);
+        if(typeof assetTypeLayerName === 'undefined'){
+          assetTypeLayerName = getSelectedAssetByTypeId(linearAssets, assetTypeId);
+        }
+        if(typeof assetTypeLayerName === 'undefined'){
+          assetTypeLayerName = getSelectedAssetByTypeId(groupedPointAssets, assetTypeId);
+        }
```

Comment:
can be shortened to        ` assetTypeLayerName = getSelectedAssetByTypeId(pointAssets, assetTypeId) || getSelectedAssetByTypeId(linearAssets, assetTypeId);
`
Grouped point assets can't be selected like that. Pointlike weight limits (TR painorajoitukset) is a layer with 4 different asset types so there needs to be a different solution if you want that as the default layer.